src/wasm/literal.cpp

```diff
@@ -1653,6 +1653,9 @@ Literal Literal::absI32x4() const {
 Literal Literal::negI8x16() const {
   return unary<16, &Literal::getLanesUI8x16, &Literal::neg>(*this);
 }
+Literal Literal::popcntI8x16() const {
+  return unary<16, &Literal::getLanesUI8x16, &Literal::popCount>(*this);
```

Comment:
Is each lane signed or unsigned? The LLVM builtin [says](https://github.com/llvm/llvm-project/blob/6335446c99eb7263b8ae89a071df8a2cd66940d5/clang/include/clang/Basic/BuiltinsWebAssembly.def#L117) it's signed though

Reply:
Signedness doesn't really make a difference for popcnt since it doesn't interpret the bits as a number. However, it's important to use the unsigned lane extraction function here because it will zero-extend each lane. If we instead used the signed lane extraction function, it would sign-extend each lane which could change the population counts.